### ISSUE TITLE
Fail to list images having more than 1 filter

### ISSUE THREAD

**Issue by renard** [pdfcpu/pdfcpu#329]
On some PDF document images can have several filters such as FlateDecode and DCTDecode.
pdfcpu skip such image with following log:

```
INFO: 2021/05/05 14:22:27 ExtractImage(72): skip img with more than 1 filter: FlateDecode,DCTDecode
```

Is there a particular reason for that? is is possible to decompress the image before applying the DCTDecode filter?

**Comment by hhrutter** [comment]
Serious support of multi filter streams need proper test cases.
I do not have any test PDFs available, that's the simple reason this has not been implemented.
Also there are a lot of possible combinations.
If you can share something we can make this work at least for the combinations we have test files for.

**Comment by renard** [comment]
I have some copyrighted material.
If you want I can share something via your private email.

Let me know.

**Comment by hhrutter** [comment]
You find my email at https://github.com/hhrutter
👍 This will help with the implementation but I also need some small files for a testcase.
Maybe there is a free PDFWriter out there that produces multifilter XObject image streams.
@petervwyatt Do you have any insights into this?

**Comment by hhrutter** [comment]
This issue is work in progress.

**Comment by hhrutter** [comment]
Fixed with v0.3.12
